project/src/main/webapp/style.css

```diff
@@ -1,9 +1,251 @@
+}
+
+/*** Landing Page ***/
+/** Workout Section **/
+.workout-header {
+    margin: 50px;
+    text-align: center;
+    font-family: fantasy;
+}
+
+.workout-selection {
+    display: flex;
+    justify-content: center;
+}
+
+.workout-options {
+    width: 1500px;
+    height: 500px;
+    padding: 50px;
+    border-radius: 25px;
+    background-image: url(https://images.unsplash.com/photo-1557564437-0995702f88fc?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1950&q=80);
+    background-position: bottom;
+}
+
+.workout-options {
+    overflow-x: auto;
+    overflow-y: hidden;
+    white-space: nowrap;
+    display: flex;
+    align-items: center;
+}
+
+.workout-options button{
+    border-color: black;
+    border-radius: 25px;
+    padding: 50px;
+    margin: 50px;
+    cursor: pointer;
+}
+
+.workout-options button:hover {
+    background-color: lightgray;
+}
+
+.workout-options button:active {
+    background-color: white;
+    transform: translateY(4px);
+}
+
+.workout-submit {
+    margin: 50px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.workout-submit label {
+    font-family: fantasy;
+    margin: 5px 10px 5px 0;
+}
+
+.workout-submit input {
+    font-family: fantasy;
+    vertical-align: middle;
+    margin: 5px 10px 5px 0;
+    padding: 10px;
+    background-color: #fff;
+    border: 1px solid #ddd;
+}
+
+/** Duration Section **/
+.duration-header {
+    margin: 50px;
+    text-align: center;
+    font-family: fantasy;
+}
+
+.duration-selection {
+    display: flex;
+    justify-content: center;
+}
+
+.duration-form {
+    display: flex;
+    flex-flow: row wrap;
+    align-items: center;
+}
+
+.duration-form label {
+    font-family: fantasy;
+    margin: 5px 10px 5px 0;
+}
+
+.duration-form input {
+    font-family: fantasy;
+    vertical-align: middle;
+    margin: 5px 10px 5px 0;
+    padding: 10px;
+    background-color: #fff;
+    border: 1px solid #ddd;
+}
+
+.final-submission {
+    margin: 100px;
+    display: flex;
+    justify-content: center;
+}
+
+.final-submission button {
+    font-family: fantasy;
+    font-size: large;
+    padding: 30px 40px;
+    background-color: dodgerblue;
+    border: 1px solid #ddd;
+    border-radius: 25px;
+    color: white;
+}
+
+.final-submission button:hover {
+    background-color: royalblue;
+    cursor: pointer;
+}
+
+/*** Playlist Page ***/
+.playlist-header {
+    margin: 50px;
+    text-align: center;
+    font-family: fantasy;
+}
+
+.recommend-playlist {
+    display: flex;
+    justify-content: center;
+}
+
+.play-playlist {
+    margin: 100px;
+    display: flex;
+    justify-content: center;
+}
+
+.play-playlist button {
+    font-family: fantasy;
+    font-size: large;
+    padding: 30px 40px;
+    background-color: dodgerblue;
+    border: 1px solid #ddd;
+    border-radius: 25px;
+    color: white;
+}
+
+.play-playlist button:hover {
+    background-color: royalblue;
+    cursor: pointer;
+}
```

Comment:
It looks like this is the same as `.final-submission`. You can use one class instead, and give both elements that class. Then, just change these three CSS rules to `.your-new-class-name`, `.your-new-class-name button`, and `.your-new-class-name button:hover`. That way, we don't duplicate code, and the size of the CSS can be reduced.